src/SixLabors.Fonts/Tables/AdvancedTypographic/FeatureTag.cs

```diff
@@ -0,0 +1,820 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.Fonts.Tables.AdvancedTypographic
+{
+    /// <summary>
+    /// Enum for the different font features.
+    /// For the complete docs, see: https://docs.microsoft.com/en-us/typography/opentype/otspec183/features_ae
+    /// </summary>
+    internal enum FeatureTag : uint
+    {
+        /// <summary>
+        /// Access All Alternates.
+        /// This feature makes all variations of a selected character accessible. This serves several purposes: An application may not support the feature by which the desired glyph would normally be accessed;
+        /// the user may need a glyph outside the context supported by the normal substitution, or the user may not know what feature produces the desired glyph.
+        /// Since many-to-one substitutions are not covered, ligatures would not appear in this table unless they were variant forms of another ligature.
+        /// </summary>
+        Aalt = 0x61616C74U,
```

Comment:
We actually have a Tag struct that's designed to hold all features, langs, and all sorts.

`Tag.Parse("aalt")` would give you a tag with the `uint` Value.